src/NinjaTools.FluentMockServer.API/Serializtion/Converters/UnwrapConverter.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using JetBrains.Annotations;
+using Newtonsoft.Json;
+
+namespace NinjaTools.FluentMockServer.API.Tests.Serializtion.Wrapping
+{
+    public class UnwrapConverter : JsonConverter
+    {
+        /// <inheritdoc />
+        public override void WriteJson(JsonWriter writer, [CanBeNull] object value, JsonSerializer serializer)
+        {
+            if (!(value is IUnwrappable wrapped))
+                return;
+
+            var target = wrapped.Unwrap();
+            var targetType = wrapped.GetInnerType();
+            serializer.Serialize(writer, target, targetType);
+        }
+
+        /// <inheritdoc />
+        [CanBeNull]
+        public override object ReadJson(JsonReader reader, Type objectType, [CanBeNull] object existingValue,  [NotNull] JsonSerializer serializer)
+        {
+            if (!typeof(IUnwrappable).IsAssignableFrom(objectType))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4802644)